GasGrid/OtherTools/Mapping and Location Tools/mapbox_CMCL/index.html

```diff
@@ -260,6 +257,43 @@
                 });
             });
 
+
+            map.addLayer({
+                id: 'terminals_name',
+                type: 'symbol',
+                source: 'terminals_s',
+                layout: {
+                    'text-field': ['get','name'],
+                    'text-font': ['DIN Offc Pro Medium', 'Arial Unicode MS Bold'],
+                    'text-size': 14,
+                    'text-offset':[0,1.5],
+                    },
+                "paint": {
+                    "text-color": "#3E3E3E",
+                    "text-halo-color": "#FFFFFF",
+                    "text-halo-width": 1,
+                    "text-opacity": ['interpolate', ['exponential',2], ['zoom'], 5, 0.3, 8, 1]
+                    }
+                });
+
+                map.addLayer({
+                id: 'offtakes_name',
+                type: 'symbol',
+                source: 'offtakes',
+                layout: {
+                    'text-field': ['get','Offtake Point (Licence Name)'],
+                    'text-font': ['DIN Offc Pro Medium', 'Arial Unicode MS Bold'],
+                    'text-size': 12,
+                    'text-offset':[0,1.5]
+                    },
+                "paint": {
+                    "text-color": "#3E3E3E",
+                    "text-halo-color": "#FFFFFF",
+                    "text-halo-width": 1,
+                    "text-opacity": ['interpolate', ['exponential',2], ['zoom'], 5, 0.3, 8, 1]
+                }
+                });
+
```

Comment:
Additional layers with text corresponding to correct property in terminals.geoJSON and offtakes.geoJSON. This is the same font as the background map font so fits in quite seamlessly with existing place names.